vulnerabilities/importers/project_kb_msr2019.py

```diff
@@ -80,15 +68,23 @@ def fetch(self):
     @staticmethod
     def to_advisories(csv_reader):
         # Project KB MSR csv file has no header row
-        advsiories = []
+        advisories = []
         for row in csv_reader:
             vuln_id, proj_home, fix_commit, _ = row
             commit_link = proj_home + "/commit/" + fix_commit
-            advsiories.append(
+
+            if re.compile(r"CVE-\d+-\d+").match(vuln_id):
```

Comment:
Using this in a loop is likely a bad idea performance and clarity-wise
I would create this instead, likely outside and in a utilities module as this is likely useful in other places:
```
is_cve = re.compile(r"CVE-\d+-\d+", re.IGNORE_CASE).match
```
then use with:
```
if is_cve(vuln_id):
....
```

Reply:
This is awesome idea !

Reply:
Done